internal/tls/config.go

```diff
@@ -130,7 +130,7 @@ func (c *ServerConfig) TLSConfig() (*tls.Config, error) {
 
 	if tlsConfig.MinVersion != 0 && tlsConfig.MaxVersion != 0 && tlsConfig.MinVersion > tlsConfig.MaxVersion {
 		return nil, fmt.Errorf(
-			"tls min version %q can't be greater then tls max version %q", tlsConfig.MinVersion, tlsConfig.MaxVersion)
+			"tls min version %q can't be greater than tls max version %q", tlsConfig.MinVersion, tlsConfig.MaxVersion)
```

Comment:
This was a "mass" change `greater then` (Google Chrome even complains about this in this comment field) => `greater than`. I didn't initially check for `less then` -- there's one instance of that as well.